#point1:center point, point2,3:vertex
int16  point1_x
int16  point1_y
int16  point2_x
int16  point2_y
int16  point3_x
int16  point3_y

---
bool success
